javascript/javascript1/week2/student-maneger.js

```diff
@@ -1,19 +1,25 @@
-let class14Students = ['maha', 'Elias', 'Anil' , 'Deepti'];
-function addStudentToClass(studentName) { 
-     if (class14Students.length >= 6) {
-       return 'no more student can be added to class 14';
-    } else if  (class14Students.includes(studentName)) {
-       return 'Student ' + studentName +  ' is already in the class'; 
-    } else if  (class14Students.length < 6 || studentName == 'queen' ) {
-        return  studentName + ' is added to class 14 as the '+
-         class14Students.push(studentName)+ 'th ' + 'student';
-    }  
+let class07Students = ['Maha', 'kamal', 'Gary', 'Karolina', 'shruthi'];
+let studentName;
+
+function addStudentToClass(studentName) {  
+if (studentName.length === 0) {   
+    return 'what is your name?';
+}else if (class07Students.includes(studentName)) {
+    return `${studentName} is already in the class`;
+}else if (class07Students.length >=6 && studentName !== 'queen') {
+    return 'Sorry but we can\'t add more students to class 15';   
+}else {
+    return `${class07Students.push(studentName)} welcome ${studentName} , you\'r added to class 15`; 
+} 
 }
 
+function getNumberOfStudents() {
+const totalNrOfStudents = addStudentToClass.length;
+  return addStudentToClass(class07Students).length;
```

Comment:
This is little bit complicated that you try to use the length property on a function which  returns a string.
instead use the global variable class07Students to get the length of the array, which gives the number of students in the class.